packages/web/components/Guild/GuildTile.tsx

```diff
@@ -26,7 +27,7 @@ export const GuildTile: React.FC<Props> = ({ guild }) => (
     _hover={{ textDecoration: 'none' }}
     href={`/guild/${guild.guildname}`}
   >
-    <MetaTile>
+    <MetaTile minW={'300px'} height="full" width="full" cursor="pointer">
```

Comment:
What happens if `minW` is omitted? It seems like `w` `full` should handle with width sufficiently.

```suggestion
    <MetaTile minW="19rem" w="full" h="full" cursor="pointer">
```

Reply:
I've left it as minW='300px'

A typical use of this is to prevent something from getting too narrow if the screen is really narrow.

In this case it seems like it would stop the tile going narrower than 300px on a super-small phone screen.

![288px no min-width Screenshot 2022-12-27 090345](https://user-images.githubusercontent.com/92962228/209597631-028f1432-b6a0-40f0-90f3-9c8783e96b6c.png)

☝ That's what happens to the layout on a 288px wide screen if the minW is removed.

Someone viewing the site on a 288px screen is not very likely to happen! My phone (for e.g.) is 320px wide, and it's about as small as you can get these days.

I will leave it in just in case. And it should be a pixel value as it's targeting a screen width, which is measured in pixels. (rem units are based on the font size of the page root element. On the MG site, that's 16px, so 19rem does equal 300px, but if the font-size was changed [again, not likely to happen!] then 19rem would no longer equal 300px)

Reply:
The 300px also seems related to this one, https://github.com/MetaFam/TheGame/issues/1474